sdk/core/Azure.Core/README.md

```diff
@@ -132,6 +132,13 @@ ConfigurationSetting setting = client.Get("Key");
 Assert.AreEqual("Value", setting.Value);
 ```
 
+### Setting up console logging
```

Comment:
I would like to have some sort of delimiter that this region is gong into a readme.

Let’s prefix the regions with `Snippet:`